From b95278d59ae18d20a2e8dabfc51b00e764ad9c22 Mon Sep 17 00:00:00 2001
From: Tony Lin <tony.lin@freescale.com>
Date: Fri, 28 Oct 2011 13:50:59 +0800
Subject: [PATCH 0503/2463] ENGR00160940-1 [MX6Q]sdhci: add delay line setting
 in board data

driver could set delay line due to board data parameter.

Signed-off-by: Tony Lin <tony.lin@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.c |    1 +
 arch/arm/plat-mxc/include/mach/esdhc.h   |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.c b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
index 9f56e92..7ef2be9 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
@@ -364,6 +364,7 @@ static const struct esdhc_platform_data mx6q_sabreauto_sd3_data __initconst = {
 	.wp_gpio = MX6Q_SABREAUTO_SD3_WP,
 	.support_18v = 1,
 	.support_8bit = 1,
+	.delay_line = 0,
 	.platform_pad_change = plt_sd3_pad_change,
 };
 
diff --git a/arch/arm/plat-mxc/include/mach/esdhc.h b/arch/arm/plat-mxc/include/mach/esdhc.h
index ee9e41c..9ef53ab 100644
--- a/arch/arm/plat-mxc/include/mach/esdhc.h
+++ b/arch/arm/plat-mxc/include/mach/esdhc.h
@@ -25,6 +25,7 @@ struct esdhc_platform_data {
 	unsigned int always_present;
 	unsigned int support_18v;
 	unsigned int support_8bit;
+	unsigned int delay_line;
 	int (*platform_pad_change)(int clock);
 };
 #endif /* __ASM_ARCH_IMX_ESDHC_H */
-- 
1.7.7.4

